packages/website/src/theme/CodeBlock/Content/styles.module.css

```diff
@@ -6,6 +6,7 @@
   /* rtl:ignore */
   direction: ltr;
   border-radius: inherit;
+  min-height: 5.5rem;
```

Comment:
Oops: https://github.com/typescript-eslint/typescript-eslint/issues/8110

cc @haocheng6 as fyi, this was my mistake, I should have caught this in review.